Dockerfile

```
@@ -1,18 +1,22 @@
-FROM ubuntu:22.04
+FROM condaforge/mambaforge:4.9.2-5 as conda
 
-ARG DEBIAN_FRONTEND=noninteractive
-RUN apt update
+COPY environment.yml .
+RUN mamba env create -f environment.yml && conda clean -afy
+RUN find -name '__pycache__' -type d -exec rm -rf '{}' '+'
+RUN rm -rf /opt/conda/envs/nmma/lib/python3.9/site-packages/pip
+RUN rm -rf /opt/conda/envs/nmma/lib/python3.9/i{dlelib, ensurepip}
+RUN rm -rf /opt/conda/envs/nmma/lib{a,t,l,u}san.so
+RUN find -name '*.a' -delete
```

You want this all in one Docker layer:
```suggestion
COPY environment.yml .
RUN mamba env create -f environment.yml && conda clean -afy && \
        find -name '__pycache__' -type d -exec rm -rf '{}' '+' && \
        rm -rf /opt/conda/envs/nmma/lib/python3.9/site-packages/pip && \
        rm -rf /opt/conda/envs/nmma/lib/python3.9/i{dlelib, ensurepip} && \
        rm -rf /opt/conda/envs/nmma/lib{a,t,l,u}san.so && \
        find -name '*.a' -delete
```